spec/api/connections_spec.cr

```diff
@@ -98,4 +98,34 @@ describe LavinMQ::HTTP::ConnectionsController do
       end
     end
   end
+
+  describe "GET /api/connections/username/:username" do
+    it "returns connections for a specific user" do
```

Comment:
Copy-paste mistake? This spec looks the same as the next one?

Reply:
hehe yes! just about to push the changes :)